NewForce-Cohort-4/nutshell-the-canine-pirates
# Articles Ticket

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
When a user registers an email and logs in with that email, they will see a button that says "Add a new article".
When a user clicks the button, they will be presented with a form to add the url, summary, and title of an article.
When a user fills out the form and clicks the save button, their info will be saved to the database, and it will print to the dom. If they add more than 1 article, their articles will print to the dom with the newest article on top.
Each article has a delete button. When the user clicks the delete button, the article info will disappear from the dom, and the info will be deleted from the database.
When a user logs out, a new user can register an email and log in with their email.
When that happens, the new user can save articles and they will appear exactly the same way as the other user. Each user will only see their articles when they are logged in.
Fixes # (issue)
## Type of change
# Testing Instructions
You should register an email. You should log in with that email. Click the add a new article button. Fill out the form and click save. See your article. Click the delete button. Watch your article vanish. Repeat as often as you wish. Log out, register a new user if desired. Repeat the steps to save articles and delete them.
# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
